resources/views/alert.blade.php

```diff
@@ -1,3 +1,10 @@
+@props([
+    'message' => null,
+    'messageClass' => 'text-sm',
+    'type' => null,
+    'large' => false,
+])
+
 <div class="alert-wrapper alert-{{ $type }}">
     <div class="alert-icon-wrapper alert-{{ $type }}-icon @if($large ?? false) alert-icon-large @endif">
```

Comment:
when adding props, you should remove all the obsolete `?? <default>` constructions